### ISSUE TITLE
Incompatibility with nbconvert 6

### ISSUE THREAD

**Issue by Cadair** [chrisjsewell/pytest-notebook#10]
When running with nbconvert 6 installed, I get

```
AttributeError: 'ExecuteCoveragePreprocessor' object has no attribute 'setup_preprocessor'
```

irrespective of the value of the coverage setting.

**Comment by chrisjsewell** [comment]
Yeh should look to move to https://github.com/jupyter/nbclient
Also https://github.com/jupyter/nbclient/pull/79 may be useful

**Comment by chrisjsewell** [comment]
Now fixed in pytest-notebook 0.7.0 (nbconvert is no longer a dependency)
